NSS-Therapeutic-Raccoons/WorkforceManagement
# updated test to get an employee from db to test with for stronger tes…

## Link to Ticket
Closes # [3](https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/3)

## Description of Proposed Changes
- Created mvc for Index of Employees.
- Created integration test for /employees to check for proper data

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rb-employee-list
```
run the tests and verify in browser


## Impacted Areas in Application

List general components of the application that this PR will affect:

* IntegrationTests/EmployeeTests.cs
    - added Get_IndexContentVerified()
    - added GetFullEmployee()
* Views/Employee/Index.cshtml
    - modified file to remove extra tds and put the fullname together.

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
